### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/WFC3/uvis_pam_corrections/WFC3_UVIS_Pixel_Area_Map_Corrections_for_Subarrays.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#304]
## What
An accessibility scan flagged 23 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/WFC3/uvis_pam_corrections/WFC3_UVIS_Pixel_Area_Map_Corrections_for_Subarrays.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell1">` (selector: `#codecell1`)
- `<pre id="codecell2">` (selector: `#codecell2`)
- `<pre id="codecell3">` (selector: `#codecell3`)
- `<table id="table140182248331088" class="table-striped table-bordered table-condensed">` (selector: `#table140182248331088`)
- `<table id="table140182511608848" class="table-striped table-bordered table-condensed">` (selector: `#table140182511608848`)
- `<table id="table140182246538000" class="table-striped table-bordered table-condensed">` (selector: `#table140182246538000`)
- `<pre id="codecell7">` (selector: `#codecell7`)
- `<pre id="codecell8"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/icjd05saq_flt.fits to ./mastDownload/HST/icjd05saq/icjd05saq_flt.fits ...
</pre>` (selector: `#codecell8`)
- `<pre id="codecell10">` (selector: `#codecell10`)
- `<pre id="codecell11"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/icw201ciq_flt.fits to ./mastDownload/HST/icw201ciq/icw201ciq_flt.fits ...
</pre>` (selector: `#codecell11`)
- `<pre id="codecell13"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/icw201d6q_flt.fits to ./mastDownload/HST/icw201d6q/icw201d6q_flt.fits ...
</pre>` (selector: `#codecell13`)
- `<pre id="codecell15"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/icw201d8q_flt.fits to ./mastDownload/HST/icw201d8q/icw201d8q_flt.fits ...
</pre>` (selector: `#codecell15`)
- `<pre id="codecell17">` (selector: `#codecell17`)
- `<pre id="codecell18"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ichz02ayq_flt.fits to ./mastDownload/HST/ichz02ayq/ichz02ayq_flt.fits ...
</pre>` (selector: `#codecell18`)
- `<pre id="codecell21">` (selector: `#codecell21`)
- `<pre id="codecell24">` (selector: `#codecell24`)
- `<pre id="codecell29">` (selector: `#codecell29`)
- `<pre id="codecell31">` (selector: `#codecell31`)
- `<pre id="codecell32">` (selector: `#codecell32`)
- `<pre id="codecell35">` (selector: `#codecell35`)
- `<pre id="codecell36">` (selector: `#codecell36`)
- `<pre id="codecell39">` (selector: `#codecell39`)
- `<pre id="codecell40">` (selector: `#codecell40`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
